blacktie/common/C/pom.xml

```diff
@@ -434,6 +434,21 @@
                                     </tasks>
                                 </configuration>
                             </execution>
+                            <execution>
+                                <id>rename libapr-1.so</id>
+                                <phase>initialize</phase>
+                                <goals>
+                                    <goal>run</goal>
+                                </goals>
+                                <configuration>
+                                    <tasks>
+                                        <echo message="rename libapr-1.so"/>
+                                        <exec executable="find">
+                                            <arg line="${project.build.directory} -name libapr-1.so -exec mv '{}' '{}'.0 ;"/>
```

Comment:
yeah, I think it would be better to create a symbol link libapr-1.so -> libapr-1.so.0 and there should be only one apr shared library in the assembly zip and tar.gz
I will update the find -exec arg to add the "ln -r -s {}.0 {}"